autosklearn/__init__.py

```diff
@@ -1,2 +1,14 @@
 # -*- encoding: utf-8 -*-
+from autosklearn.util import dependencies
+
 __version__ = '0.1.0'
+
+__MANDATORY_PACKAGES__ = '''
+scikit-learn==0.17.1
+smac==0.0.1
+lockfile>=0.10
+ConfigSpace>=0.2.1
+pyrfr==0.2.0
+'''
+
+dependencies.verify_packages(__MANDATORY_PACKAGES__)
```

Comment:
Can you move this to `automl.py`? I think it is the only file that will always be imported.